services/storage/src/simcore_service_storage/modules/celery/models.py

```diff
@@ -0,0 +1,15 @@
+from typing import Any, TypeAlias
+from uuid import UUID
+
+from models_library.progress_bar import ProgressReport
+from pydantic import BaseModel
+
+TaskContext: TypeAlias = dict[str, Any]
+TaskID: TypeAlias = str
+TaskUUID: TypeAlias = UUID
+
+
+class TaskStatus(BaseModel):
+    task_uuid: TaskUUID
+    task_state: str  # add enum
+    progress_report: ProgressReport | None = None
```

Comment:
I would always expect to have a proress report. Put a default to progress (0) isntead of being None